spec/datadog/core/configuration/agent_settings_resolver_spec.rb

```diff
@@ -22,15 +23,15 @@
       hostname: hostname,
       port: port,
       uds_path: uds_path,
-      timeout_seconds: nil,
-      deprecated_for_removal_transport_configuration_proc: nil,
+      timeout_seconds: timeout_seconds
     }
   end
 
   let(:adapter) { :net_http }
   let(:hostname) { '127.0.0.1' }
   let(:port) { 8126 }
   let(:uds_path) { nil }
+  let(:timeout_seconds) { 30 }
 
   before do
     # Environment does not have existing unix socket for the base testing case
```

Comment:
:arrow_down: the " 'when settings.tracing is not present'" block below can now be removed as it was protecting against the tracing section of the settings not being present, e.g. this bit that got removed:

![image](https://github.com/DataDog/dd-trace-rb/assets/2785847/df561c3a-c678-4045-802f-5a3bbdd0ee4c)